examples/device/cdc_dual_ports/src/usb_descriptors.c

```diff
@@ -93,8 +93,8 @@ enum
 
 uint8_t const desc_configuration[] =
 {
-  // Interface count, string index, total length, attribute, power in mA
-  TUD_CONFIG_DESCRIPTOR(ITF_NUM_TOTAL, 0, CONFIG_TOTAL_LEN, TUSB_DESC_CONFIG_ATT_REMOTE_WAKEUP, 100),
+  // Interface count, string index, total length, attribute, power in mA, config number
+  TUD_CONFIG_DESCRIPTOR(ITF_NUM_TOTAL, 0, CONFIG_TOTAL_LEN, TUSB_DESC_CONFIG_ATT_REMOTE_WAKEUP, 100, 1),
```

Comment:
the confi_number should be the 1st parameter though since it makes more sense that way. Even though it could break current usage. Most larger project use their own descriptor, smaller project should be easily change to this. Luckily I has just release 0.6.0 after like 1000 commit after 0.5.0 :). We can document it in changelog in migration section for next 0.7.0 release.